### DISCUSSION TITLE
[Feature Request] Nvidia GeForce 256 DDR card emulation

### DISCUSSION THREAD

**Discussion by mrmaller1905** [86Box/86Box#4673] [General]
Can you add support for emulation of Nvidia's GeForce 256 DDR to 86Box?

Thanks in advance.

**Comment by TC1995** [comment]
no

**Comment by eddmanx** [comment]
There aren't even register level documents available to emulate a TNT, let alone a geforce 256. (Regular whitepapers and spec sheet PDFs that you see around the internet don't have that info)

Even if such documents were somehow found, it's very much doubtful that current or near future upcoming CPUs could emulate it at 100% speed.

**Comment by ppgrainbow** [comment]
Not going to happen. The nVIDIA GeForce 256 DDR has 32 MB of video RAM and I don't think that any CPUs will be able to emulate it at 100% speed. There are no whitepaper and specification sheets for these such graphics cards and they lack of register level documentations that are designed to emulate a a pre-GeForce-based nVIDIA graphics card.

I would suggest that you do some more research BEFORE suggesting such video cards.

**Comment by LeoKids** [comment]
TNT2,TNT and Riva 128 is fine to emulate on 86box,the 256 is not. I'm afraid not even the budget models will be added here.

**Reply by jriwanek** [reply]
That is incorrect, we don't have the required datasheets, and therefore cannot correctly emulate those cards at this time, if however you know something we don't, we look forward to your well documented PR.

**Reply by mrmaller1905** [reply]
> That is incorrect, we don't have the required datasheets, and therefore cannot correctly emulate those cards at this time, if however you know something we don't, we look forward to your well documented PR.

Will 86Box add support for Riva 128 and TNT emulation in the future versions/builds?

**Reply by MrPepka** [reply]
Read all the comments in this discussion carefully and you will find the answer to this question

**Reply by LeoKids** [reply]
Sorry,but i was thinking of probability,not actual possibility.

**Comment by angelosa** [comment]
fwiw all ELSA GeForce 256/Quadro cards will also perform a protection check at POST time, which makes this an uphill battle right off the bat.
![elsa_prot](https://github.com/user-attachments/assets/104eaad7-70f9-4260-9842-7bd8f2139456)

**Comment by Torinde** [comment]
https://github.com/joncampbell123/dosbox-x/issues/3412#issuecomment-2743227563

> Some related effort could be found here:
https://github.com/bochs-emu/Bochs/discussions/496
https://github.com/bochs-emu/Bochs/compare/master...Vort:Bochs:geforce

**Reply by sskras** [reply]
Until now this effort aimed at 2D only and the following models:

* GeForce 3 (NV20)
* GeForce FX 5600 (NV35)
* GeForce 6800 (NV40)

But recently his account was seemingly compromised: https://github.com/Vort/Bochs/issues/4#issuecomment-3005523581 : (
